### PR TITLE
Add DynamoDB config file support

### PR DESCRIPTION
**Pull request by brandonstrohmeyer**
Add support to filter DynamoDB tables by name in config file.

Test output:
https://gist.github.com/brandonstrohmeyer/5db0fe02483b35c0900f13f2c876b301

### CODE REVIEW

File: aws/dynamodb_test.go [hunk 1]
```diff
@@ -81,7 +82,7 @@ func TestGetTablesDynamo(t *testing.T) {
 	)
 
 	require.NoError(t, err)
-	getAllDynamoTables(awsSession, time.Now().Add(1*time.Hour*-1), db)
+	getAllDynamoTables(awsSession, time.Now().Add(1*time.Hour*-1), config.Config{}, db)
```
Comment (denis256) [inline aws/dynamodb_test.go:85]: Result of function invocation not checked, I think at least should be checked for errors

File: aws/dynamodb.go [hunk 2]
```diff
@@ -61,6 +59,22 @@ func getAllDynamoTables(session *session.Session, excludeAfter time.Time, db Dyn
 	return tableNames, nil
 }
 
+func shouldIncludeTable(table *dynamodb.TableDescription, excludeAfter time.Time, configObj config.Config) bool{
```
Comment (denis256) [inline aws/dynamodb.go:62]: `shouldIncludeTable` is not covered by tests from `aws/dynamodb_test.go`

image

[coverage.html.zip](https://github.com/gruntwork-io/cloud-nuke/files/7691179/coverage.html.zip)

File: aws/dynamodb_test.go (hunk 1 above)
Comment (brandonstrohmeyer) [reply aws/dynamodb_test.go]: Fixed

File: aws/dynamodb.go (hunk 2 above)
Comment (brandonstrohmeyer) [reply aws/dynamodb.go]: Added new test for function:

❯ go test -v -run TestShouldIncludeTable
=== RUN   TestShouldIncludeTable
=== RUN   TestShouldIncludeTable/ConfigExclude
=== RUN   TestShouldIncludeTable/ConfigInclude
=== RUN   TestShouldIncludeTable/NotOlderThan
--- PASS: TestShouldIncludeTable (0.00s)
    --- PASS: TestShouldIncludeTable/ConfigExclude (0.00s)
    --- PASS: TestShouldIncludeTable/ConfigInclude (0.00s)
    --- PASS: TestShouldIncludeTable/NotOlderThan (0.00s)
PASS
ok      github.com/gruntwork-io/cloud-nuke/aws  0.201s

**Review by denis256 — APPROVED** [review]

**Review by denis256 — APPROVED** [review]

**Comment by yorinasub17** [comment]
LGTM as well! I'll kick off a sanity check regression build now and if that passes, we can merge this in! Thanks for your contribution!

**Comment by yorinasub17** [comment]
Build passed, so will merge this in! Thanks again for your contribution!

**Pull request merged**